game/lib/profile_page/achievement_cell.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:game/utils/utility_functions.dart';
+
+/**
+ * Widget that represents each individual achievement
+ * @param name: Name of the achievement
+ * @param tasksFinished: Number of currently completed tasks towards the achievement
+ * @param totalTasks: Total number of tasks needed to gain achievement
+ * @param picture: picture that is associated with the achievement
+ */
+Widget achievementCell(
```

Comment:
We may not end up including achievements in our MVP - at the moment it's a low priority feature, but this code is a good template for future features/designs we implement.

Reply:
Although we can probably still include it for now. Easier to hide something from our MVP than to rebuild a page later.